providers/edu/harvarddataverse/normalizer.py

```diff
@@ -8,34 +8,24 @@ class Person(Parser):
     suffix = ParseName(ctx).suffix
 
 
-class Contributor(Parser):
-    person = Delegate(Person, ctx)
-    cited_name = ctx
+class Creator(Parser):
+    agent = Delegate(Person, ctx)
     order_cited = ctx('index')
+    cited_as = ParseName(ctx).last
 
 
-class Link(Parser):
-    url = ctx
-    type = RunPython('get_link_type', ctx)
-
-    def get_link_type(self, link):
-        if 'dx.doi.org' in link:
-            return 'doi'
-        elif 'dataverse.harvard.edu' in link:
-            return 'provider'
-        return 'misc'
-
-
-class ThroughLinks(Parser):
-    link = Delegate(Link, ctx)
+class WorkIdentifier(Parser):
+    uri = IRI(ctx)
 
 
 class CreativeWork(Parser):
+    schema = 'dataset'
     title = ctx.name
     description = Try(ctx.description)
-    contributors = Map(Delegate(Contributor), Try(ctx.authors))
     date_published = ParseDate(ctx.published_at)
-    links = Concat(
-        Delegate(ThroughLinks, ctx.url),
-        Delegate(ThroughLinks, ctx.image_url),
-    )
+    identifiers = Map(Delegate(WorkIdentifier), ctx.url)
```

Comment:
Throw `global_id` in as well, just for good measure. It will be de-duped if necessary